package.json

```diff
@@ -17,15 +17,18 @@
     "express": "^4.17.1",
     "express-validator": "^6.6.1",
     "nodemon": "^2.0.6",
+    "npm-run-all": "^4.1.5",
     "sqlite3": "^5.0.0"
   },
   "scripts": {
-    "prebuild": "rm -rf dist/* && tslint -c tslint.json -p tsconfig.json --fix",
-    "build": "tsc && cp -rf package.json dist/package.json && cp -rf database.db dist/database.db",
-    "prestart": "npm run build",
-    "start": "nodemon .",
-    "debug": "yarn build && nodemon --inspect .",
     "migration:run": "ts-node ./node_modules/typeorm/cli.js migration:run",
-    "migration:revert": "ts-node ./node_modules/typeorm/cli.js migration:revert"
+    "migration:revert": "ts-node ./node_modules/typeorm/cli.js migration:revert",
+    "dev": "nodemon --config nodemon.json --watch src -e ts --exec yarn dev:start",
+    "dev:start": "npm-run-all build start",
+    "build": "npm-run-all clean lint tsc",
```

Comment:
Looks like some of the scripts use yarn while others rely on npm. This means your solution is dependent on both package managers which may lead to problems with the scripts' compatibility. My non-insistent suggestion would be to select one package manager and use it for all needs.